app/api/helpers/order.py

```diff
@@ -213,5 +223,5 @@ def calculate_order_amount(tickets, discount_code):
             'ticket_fee': round(ticket_fee, 2),
             'sub_total': round(sub_total, 2)
         })
-    return dict(tax_included=tax_included, total_amount=round(total_amount, 2), total_tax=round(total_tax, 2),
-                total_discount=round(total_discount, 2), tickets=ticket_list)
+    return jsonify(dict(tax_included=tax_included, total_amount=round(total_amount, 2), total_tax=round(total_tax, 2),
```

Comment:
This should not return JSON response. This function should have no idea about request or json or anything. This function does not know that it is part of flask application. This should be a pure python function, otherwise what was the point of extracting it out?